README.md

```diff
@@ -59,6 +67,47 @@ EXAMPLES
 
 _See code: [@oclif/plugin-autocomplete](https://github.com/oclif/plugin-autocomplete/blob/v0.3.0/src/commands/autocomplete/index.ts)_
 
+## `tart delete DELETEFILE`
+
+delete a local database dump
+
+```
+USAGE
+  $ tart delete DELETEFILE
+
+ARGUMENTS
+  DELETEFILE  name of file to delete
+
+OPTIONS
+  -c, --config=config  [default: ./tart.config.json] Path to configuration file
+
+EXAMPLES
+  $ tart delete myDB
+  $ tart delete myDB@1.0.0
+```
+
+_See code: [src/commands/delete.ts](https://github.com/sdelements/tart/blob/v0.0.0/src/commands/delete.ts)_
+
+## `tart fetch FILE`
+
+fetch specified dump file from a remote repository
+
+```
+USAGE
+  $ tart fetch FILE
+
+ARGUMENTS
+  FILE  name of the file
+
+OPTIONS
+  -c, --config=config  [default: ./tart.config.json] Path to configuration file
+
+EXAMPLE
+  $ tart fetch filename
```

Comment:
Might want to use the name@1.0.0 format here (and in all the other repo related examples) since it's enforced right now.

Reply:
We auto-generated this file and this was taken from the `fetch` command so we should probably update it there. I think I'll remove the README and we can punt the ordering and other fixes until later